File: src/main/java/land/leets/domain/interview/presentation/dto/req/InterviewRequest.java
package land.leets.domain.interview.presentation.dto.req;

import jakarta.validation.constraints.NotNull;
import lombok.AllArgsConstructor;
import lombok.Getter;

import java.time.LocalDateTime;

@Getter
@AllArgsConstructor
public class InterviewRequest {
    @NotNull
    private Long applicationId;

    @NotNull
    private LocalDateTime fixedInterviewDate;

    @NotNull
    private String place;
}

File: src/main/kotlin/land/leets/domain/interview/presentation/dto/req/InterviewRequest.kt
package land.leets.domain.interview.presentation.dto.req

import jakarta.validation.constraints.NotNull
import java.time.LocalDateTime

data class InterviewRequest(
    @field:NotNull
    val applicationId: Long,

    @field:NotNull
    val fixedInterviewDate: LocalDateTime,

    @field:NotNull
    val place: String
)
